src/sse/chainweb-data-height-tracker.ts

```diff
@@ -0,0 +1,106 @@
+import { getChainwebDataEvents } from './chainweb-data.js';
+import { validateType } from './types.js';
+import Logger from './logger.js';
+import config from '../config/index.js';
+
+const {
+  chainwebDataHeightUpdateInterval: INTERVAL_MS,
+} = config;
+
+const CLASS_NAME = 'ChainwebDataHeightTracker';
+
+type NumericCallback = (n: number) => void;
+
+export default class ChainwebDataHeightTracker {
+  private _maxHeight: number = -1;
+  private _running: boolean = false;
+  private _logger = new Logger('ChainwebDataHeightTracker');
```

Comment:
I see you use a custom logger solution. In general we use [debug](https://www.npmjs.com/package/debug) in other packages. Worth considering.

Reply:
We can discuss this further later on but as a note for this decision: I looked for a bit more advanced functionality in a few common packages (including debug) but after not finding something off the shelf that fit my requirements I built a small custom thing over npmlog

The things I was looking for:
- prefix log line with timestamp (now customizable/optional)
- prefix log with "section/module" that emitted that log line
- easily configurable log levels (error/warn/log/debug)